### PR TITLE
Create JSON and Fixtures for Section 3G

### PR DESCRIPTION
**Pull request by andrewadcock**
Create fixtures and JSON for section 3G

Currently, the information-only fieldsets do not utilize the `fieldset_type` parameter. If we think that's bad execution, I propose we add "informational" to the fieldset_type enum. This will better utilize fieldset.js but it makes the code more verbose. The trade-off seems fairly even but I'm open to discussion.

Tickets: #356 #468

### CODE REVIEW

**Review by jgillis1 — APPROVED** [review]
I added notes to the Build ticket (#211) for components that need to be built out for this section. The JSON matches the updated content document.

**Review by cl-adavyd — APPROVED** [review]

**Comment by jgillis1** [comment]
QA Review has been completed and the PR is ready for 18F review.

File: docs/section-schemas/2020-ma-section-3.json [hunk 1]
```diff
@@ -3738,8 +3738,501 @@
...
+              "show_if_state_program_type_in": [
+                "separate_chip",
+                "combo"
+              ]
+            },
+            "questions": [
+              {
+                "type": "fieldset",
+                "label": "Note on age groups",
+                "hint": "Children should be in age groups based on their age on September 30th, the end of the federal fiscal year (FFY). For example, if a child turns three years old on September 15th, the child should be included in the “ages 3-5'' group. Even if the child received dental services on September 1st while they were still two years old, all dental services should be counted as their age at the end of the FFY.",
```
Comment (mgwalker) [inline docs/section-schemas/2020-ma-section-3.json:3758]: Nice!

I thought I'd fixed how this displays in #621, but apparently not. I'll get a PR up shortly. The issue is in `CMSLegend` – if the thing being displayed doesn't have an ID, the legend is empty. Fieldsets don't have IDs, so... 😬

**Review by mgwalker — APPROVED** [review]
This is soooo much easier to reason about. I should be able to knock out #315 quickly with this structure.

I'll also get the fix up for the fieldset labels. I tried it out locally and it looks like a pretty good stopgap.

**Pull request merged**